packages/core/src/lambda/vue/remoteInvoke/invokeLambda.ts

```diff
@@ -443,22 +436,164 @@ export class RemoteInvokeWebview extends VueWebview {
     }
 
     public async listRemoteTestEvents(functionArn: string, region: string): Promise<string[]> {
-        const params: SamCliRemoteTestEventsParameters = {
-            functionArn: functionArn,
-            operation: TestEventsOperation.List,
-            region: region,
+        try {
+            const params: SamCliRemoteTestEventsParameters = {
+                functionArn: functionArn,
+                operation: TestEventsOperation.List,
+                region: region,
+            }
+            const result = await this.remoteTestEvents(params)
+            return result.split('\n').filter((event) => event.trim() !== '')
+        } catch (error) {
+            // Suppress "lambda-testevent-schemas registry not found" error - this is normal when no test events exist
+            const errorMessage = error instanceof Error ? error.message : String(error)
+            if (
+                errorMessage.includes('lambda-testevent-schemas registry not found') ||
+                errorMessage.includes('There are no saved events')
+            ) {
+                getLogger().debug('No remote test events found for function (this is normal): %s', functionArn)
+                return []
```

Comment:
Could we simplify the message by removing the '(this is normal)' part? The context makes it clear this is an expected state.